test/integration/production/test/index.test.js

```diff
@@ -61,6 +61,21 @@ describe('Production Usage', () => {
       expect(res.status).toBe(404)
     })
 
+    it('should render 501 if the HTTP method is not GET or HEAD', async () => {
+      const url = `http://localhost:${appPort}/_next/abcdef`
+      const methods = ['POST', 'PUT', 'DELETE']
```

Comment:
one minor thing: should we also add `OPTIONS` 😅

Reply:
Doesn't really matter as it tests the functionality at hand 🕵️